jib-core/src/main/java/com/google/cloud/tools/jib/registry/AuthenticationMethodRetriever.java

```diff
@@ -55,12 +57,11 @@ public BlobHttpContent getContent() {
    * The request did not error, meaning that the registry does not require authentication.
    *
    * @param response ignored
-   * @return {@code null}
+   * @return the authenticator
```

Comment:
Should we be more specific here since this isn't really returning anything?

Reply:
Oh, oops. Fixed the Javadoc.